string filename
uint16[] x
uint16[] y
uint16[] area_length
---
int16 height
